drivers/builtin/include/mbedtls/platform.h

```diff
@@ -446,6 +446,32 @@ mbedtls_platform_context;
 #include "platform_alt.h"
 #endif /* !MBEDTLS_PLATFORM_SETUP_TEARDOWN_ALT */
 
+#if !defined(MBEDTLS_ENTROPY_HARDWARE_POLL_MIN)
+#define MBEDTLS_ENTROPY_HARDWARE_POLL_MIN     32     /**< Minimum for the hardware source */
+#endif
+
+#if defined(MBEDTLS_ENTROPY_HARDWARE_POLL)
+/**
+ * \brief           User defined callback function that is used from the entropy
+ *                  module to gather entropy data from some hardware device.
+ *
+ * \warning         This is not provided by Mbed TLS.
```

Comment:
I don't think “not provided by Mbed TLS” is a useful warning. Also, it's TF-PSA-Crypto now.

On the other hand, the function's documentation should state that this function is only used when `MBEDTLS_ENTROPY_HARDWARE_POLL` is defined. This is otherwise not visible when you read the Doxygen-rendered documentation.